.github/workflows/docker-backend.yml

```
@@ -6,22 +6,17 @@ on:
         type: string
         default: docker.nexus.famedly.de
```

I think it could be good to add a comment here to be extra clear, even if this is documented in notion

```suggestion
        # docker.nexus.famedly.de is the private container registry
        default: docker.nexus.famedly.de
```